api/Dockerfile

```diff
@@ -135,12 +135,20 @@ CMD ["php-fpm"]
 # depends on the "php" stage above
 FROM caddy:${CADDY_VERSION}-builder-alpine AS api_platform_caddy_builder
 
+# renovate: datasource=go depName=github.com/dunglas/mercure
+ARG MERCURE_VERSION=v0.14.5
+# renovate: datasource=go depName=github.com/dunglas/mercure/caddy
+ARG MERCURE_CADDY_VERSION=v0.14.5
+# renovate: datasource=go depName=github.com/dunglas/vulcain
+ARG VULCAIN_VERSION=v0.4.2
+# renovate: datasource=go depName=github.com/dunglas/vulcain/caddy
+ARG VULCAIN_CADDY_VERSION=v0.4.2
 # install Mercure and Vulcain modules
 RUN xcaddy build \
-	--with github.com/dunglas/mercure \
-	--with github.com/dunglas/mercure/caddy \
-	--with github.com/dunglas/vulcain \
-	--with github.com/dunglas/vulcain/caddy
+	--with github.com/dunglas/mercure@$MERCURE_VERSION \
```

Comment:
`github.com/dunglas/mercure` and `github.com/dunglas/vulcain` is obsolete in the meantime. It's seems sufficient to only import the caddy modules.

See PR #3255

https://github.com/ecamp/ecamp3/pull/3255/files#diff-21ee93e31c9cec7a5f33b680622da377c451b62b6c44eac6d9550eade41beb47L140

Reply:
done